app/assets/javascripts/ui_components/modal.coffee

```diff
@@ -0,0 +1,13 @@
+$ ->
+  _.each $('.ui-components-modal'), (modal) ->
+    $modal = $ modal
+    trigger = $($modal.attr('data-trigger'))
+    return if trigger.length == 0
+    $(trigger).click (e) ->
+      url = $modal.attr('data-url')
+      if url && url.length > 0
+        $modal.find('.modal-body').load url, ->
+          $modal.one('shown.bs.modal', (e) -> $modal.trigger('uic:domchange'))
```

Comment:
So what? Actually it would be nice if stuff is already rendered before the modal shows. And looking at `modal.js` in twbs, it doesn't do anything with the contents.